app/views/developers/index.html.erb

```diff
@@ -41,15 +41,21 @@
 
     <!-- Paginated list of developers -->
     <div class="col-span-12 md:col-span-9">
-      <%= render Developers::CardComponent.with_collection(@query.featured_records, featured: true) %>
-
-      <%= render PagyPaginatorComponent.new(id: "developers", pagy: @query.pagy, url_array: [:developers], container_classes: "divide-y divide-gray-200", options: @query.filters) do |component| %>
-        <% component.loading_icon do %>
-          <div class="py-4">
-            <%= render LoadingComponent.new %>
-          </div>
+      <% if @query.featured_records.any? || @query.records.any? %>
+        <%= render Developers::CardComponent.with_collection(@query.featured_records, featured: true) %>
+
+        <%= render PagyPaginatorComponent.new(id: "developers", pagy: @query.pagy, url_array: [:developers], container_classes: "divide-y divide-gray-200", options: @query.filters) do |component| %>
+          <% component.loading_icon do %>
+            <div class="py-4">
+              <%= render LoadingComponent.new %>
+            </div>
+          <% end %>
+          <%= render Developers::CardComponent.with_collection(@query.records) %>
         <% end %>
-        <%= render Developers::CardComponent.with_collection(@query.records) %>
+      <% else %>
+        <div class="mt-8">
```

Comment:
```suggestion
        <div class="my-16">
```

Reply:
Your suggestion of `my-16` removed all the margin 😬 I changed it to `mt-16`.